exp/aar.md

```diff
@@ -1,24 +1,25 @@
-The Anbox Application Registry, or *AAR*, provides a central repository for applications created on Anbox Cloud. Using an AAR is very useful for larger deployments involving multiple regions, in order to keep applications in sync.
+The Anbox Application Registry (AAR) provides a central repository for applications created on Anbox Cloud. Using an AAR is very useful for larger deployments involving multiple regions, in order to keep applications in sync.
 
 <a name="aar-roles"></a>
-## How does it work
+## How AAR works
 
-You can configure the AMS service to regularly look at an AAR and import any new application versions found there. When a new application version is found, AMS starts to import it. Once done, AMS makes it available for use.
+You can configure the Anbox Management Service (AMS) to regularly check an AAR and import any new application versions found there. When a new application version is found, AMS starts to import it. Once done, AMS makes it available for use.
 
 An imported application is immutable and cannot be changed other than through the AAR itself. If an application is removed from the AAR, it is removed from AMS on the next update as well.
 
-AMS can act in two different roles when working with the AAR:
+AMS can act in two different roles, `publisher` or `client` when working with the AAR. A single AMS instance can act either as a publisher or as a client, but not both.
 
-* `publisher` - should be related to one AMS unit
-* `client` - can be related to many units
+* `publisher`:
 
-The `publisher` role allows both read and write access to the AAR. AMS instances registered as publishers act in `push` mode and are meant to push new applications and updates of these to the AAR so that they can be consumed by regular read-only `clients`.
+    The `publisher` role allows both read and write access to the AAR. AMS instances registered as publishers act in `push` mode and are meant to push new applications and their updates to the AAR so that they can be consumed by regular read-only `clients`.
 
-A single AMS instance can act only as a publisher or as a client, but not both. We recommend you have one publishing AMS instance per architecture, for example, one for `amd64` and one for `arm64`. These should not be used to host regular containers but only to manage applications. Regular users should be directed to AMS instances acting in `client` mode.
+    We recommend that you have one publishing AMS instance per architecture, for example, one for `amd64` and one for `arm64`. The publishing instances should not be used to host regular containers but only to manage applications. Regular users should be directed to AMS instances acting as `client`.
```

Comment:
I have removed the code formatting as I don't think every instance of this needs it. Especially not the words which are explaining the scenario rather than indicating the mode itself.